test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/DefaultEditorTemplatesTest.cs

```diff
@@ -822,16 +824,17 @@ public void Editor_InputTypeDateTime_RendersAsDateTime()
         // Html5DateRenderingMode.Rfc3339 is enabled by default.
         [Theory]
         [InlineData(null, null, "2000-01-02T03:04:05.060-05:00", "text")]
-        [InlineData("date", "{0:d}", "2000-01-02", "date")]
         [InlineData("datetime", null, "2000-01-02T03:04:05.060", "datetime-local")]
         [InlineData("datetime-local", null, "2000-01-02T03:04:05.060", "datetime-local")]
-        [InlineData("DateTimeOffset", "{0:o}", "2000-01-02T03:04:05.060-05:00", "text")]
-        [InlineData("time", "{0:t}", "03:04:05.060", "time")]
-        [InlineData("month", "{0:yyyy-MM}", "2000-01", "month")]
```

Comment:
We know RFC 3339 mode doesn't affect the `"month"` type, especially with a specified format. But, why remove the check here? I expected two `"week"` additions plus `[InlineData("month", null, ...)]` here and for `Editor_FindsCorrectDateOrTimeTemplate_WithTimeOffset_NotRfc3339(...)`.